ui/lib/replication/addon/templates/mode/secondaries/add.hbs

```diff
@@ -18,11 +18,14 @@
     </p>
   </div>
   <div class="field">
-    {{!-- TODO fix so it defaults to 30s like in other places or replace with new TTL picker --}}
-    {{ttl-picker onChange=(action (mut ttl)) initialValue="30m" class="is-marginless"}}
-    <p class="help has-text-grey">
-      This is the Time To Live for the generated secondary token. After this period, the generated token will no longer be valid.
-    </p>
+    <TtlPicker2
+      @initialValue="30m"
```

Comment:
![image](https://user-images.githubusercontent.com/903288/86065972-c206c000-ba25-11ea-9f53-437e0ccd6e4a.png)

![image](https://user-images.githubusercontent.com/903288/86066010-d8148080-ba25-11ea-897e-0ab2661ef7ca.png)

Judging from the code here I'd expect to see the initial value as 30 minutes, but for some reason I'm seeing 1800 seconds in the UI. I know these values are equal, but everywhere else in the UI we default to "minutes" as the unit. If possible it'd be great to fix this so it's consistent.

Reply:
This is expected behavior for the current design, but not ideal so I went ahead and updated it (with test coverage!) so that the unit matches what is passed in on the value

Reply:
I updated the TTL picker UI to match the unit passed in, but the unit being passed to the API is being standardized to seconds for a couple reasons:

1. Some of the APIs expect units, and others don't (defaulting to seconds), so the new TTL picker was designed to account for either of those
2. None of the APIs accept 'days' as a unit, but we provide that as a dropdown option, so always converting to seconds when we send to the API ensures that we are consistent at that level.

Reply:
awesome, thank you for tracking that down! shoot, i found another bug:

1. toggle the TTL picker
2. enter a value (3 minutes)
3. toggle the TTL picker off
4. the value from step 2 still gets sent to the API 🙈